packages/server-core/src/media/file-browser-upload/file-browser-upload.hooks.ts

```diff
@@ -49,6 +51,26 @@ import verifyScope from '../../hooks/verify-scope'
 //   return context
 // }
 
+function isValidFileType(fileType: string, fileName: string): boolean {
```

Comment:
We have a bunch of these validation functions all over the place, in the future it would be good to consolidate them into packages/common so they can be consumed isomorphically in both the client and server.